galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -2006,6 +2006,22 @@ Galera Cluster ships with a number of default scripts that the node can use in s
      [client]
      socket=/path/to/socket
 
+- ``xtrabackup-v2`` This option is a fast and practically non-blocking state transfer method based on the Percona ``xtrabackup-v2`` tool. If you want to use it, the following settings must be present in the ``my.cnf`` configuration file on all nodes:
+
+  .. code-block:: ini
+
+     [mysqld]
+     wsrep_sst_auth=YOUR_SST_USER:YOUR_SST_PASSWORD
+     wsrep_sst_method=xtrabackup-v2
+     datadir=/path/to/datadir
+
+     [client]
+     socket=/path/to/socket
+
+- ``wsrep_recover`` See :ref:`Crash Recovery <crash-recovery>`.
+
+- ``clone`` See :ref:`Enabling SSL for clone based SST <ssl-clone>`. Available as of MySQL database server version 8.0.
```

Comment:
also in our 8.0 and greater, we are focused solely on clone being the fastest method